source/skins/simple/raw/FormFieldSkin.js

```diff
@@ -21,6 +21,9 @@ export default themr(FORM_FIELD)((props) => (
         {props.label}
       </label>
     )}
-    {props.input}
+    <div className={props.theme.inputWrapper}>
+      {props.input}
+      {props.children}
```

Comment:
Could you elaborate on this change?

Reply:
This wraps the action input in a container, allowing item's to be positioned relatively to the actual input, as opposed to the FormField as a whole